### DISCUSSION TITLE
唉，开源有国界啊。

### DISCUSSION THREAD

**Discussion by nianyi778** [web-infra-dev/modern.js#6466] [General]
<img width="845" alt="image" src="https://github.com/user-attachments/assets/91c90b55-4a94-4a87-bc6a-300aa77620ce">

**Comment by nianyi778** [comment]
https://www.reddit.com/r/reactjs/comments/144i4me/first_impressions_of_using_modernjs/

**Comment by lanmingle** [comment]
不是很正常么？都是为了搞钱，别整天拿着开源来扯犊子，当到某些条件才会选择开源或互帮互助，字节还是那样子，比如汽水音乐，一开始说着永久免费，后面搞了个VIP，到现在还搞了VIP等级（超级会员），习惯就好，资本的世界你我都应该明白，都有代价的，哪有免费的午餐。

**Comment by chenjiahan** [comment]
This topic will be locked because it violates the Modern.js [Code of Conduct](https://github.com/web-infra-dev/modern.js/blob/main/CODE_OF_CONDUCT.md)
